exe/execution_wrapper.cc

```diff
@@ -52,10 +61,11 @@ int main(int argc, char *argv[])
 #ifndef _WIN32
         fs::permissions(params.get<std::string>("stderr"), FILE_PERMS);
 #endif
-        ofs << lth_loc::format("Task '{1}' failed to run: {2}", task_executable, e.what());
+        ofs << lth_loc::format("Executable '{1}' failed to run: {2}", executable, e.what());
         exitcode = 127;
     }
 
+    // Write the exit code; at this point, stdout and stderr are already written
 #ifdef _WIN32
     lth_file::atomic_write_to_file(std::to_string(exitcode), params.get<std::string>("exitcode"));
 #else
```

Comment:
I haven't changed this behavior in this PR, but I'm wondering why there is a write to the exitcode file here in the task wrapper, but also a PID callback that writes to the same file on the task module's call to `leatherman::execution::execute`. But I notice that if I remove the callback from the task module, some unit tests do fail. I haven't included a similar PID callback for the command module here, so I'm just relying on the task wrapper to write the exit code. I'm curious if there's something I'm missing here.

Reply:
Why do you say they write to the same file? I see the task module writing out a `pid` file (which would be the process ID of the process it just started). The task wrapper writes out a `exitcode` file when it's done (which is the exit code of that same process).

Reply:
Sorry, you're right, I was totally confused when I commented - I'll work on resolving the rest of the failures here today